### DISCUSSION TITLE
Is there any way to sponsor this project?

### DISCUSSION THREAD

**Discussion by helpermethod** [komapper/komapper#905] [Q&A]
This is such an amazing, high quality project, that it's a shame there is no way (?) to sponsor it. Did you think about Github Sponsors or Open Collective?

**Comment by nakamura-to** [comment]
Thank you for asking this question. We are currently considering using Open Collective.

**Reply by nakamura-to** [reply]
We have opened an Open Collective account.
https://opencollective.com/komapper

You can get to that page from the README link as well.
https://github.com/komapper/komapper#backers--sponsors
